cfme/utils/appliance/__init__.py

```diff
@@ -1851,6 +1851,81 @@ def fqdn(self):
         """
         return self.rest_api.get_entity_by_href(self.rest_api.server_info['server_href']).hostname
 
+    def get_disabled_regions(self, provider=None):
+        """Fetch appliance advanced config, get disabled regions for given provider's type
+
+        Only relevant for cloud providers azure and ec2 at the moment
+
+        Args:
+            provider: A BaseProvider object with settings_key attribute
+
+        Returns:
+            Default: Dict of ems_<provider> keys and values of disabled_regions map
+            when provider given: disabled_regions list from config
+            when no matching config found: None
+        """
+        ems_config = self.get_yaml_config()['ems']
+        if provider:
+            try:
+                prov_config = ems_config.get(getattr(provider, 'settings_key', None), {})  # safe
+                regions = prov_config['disabled_regions']  # KeyError
+            except KeyError:
+                logger.warning('Provider settings_key "%s" or "disabled_regions" '
+                               'not found in config %s', provider.settings_key, ems_config)
+                return None
```

Comment:
**REQUIRED** returning none here unchecked caused the tests to fail either we make this return value uniform, or handle it in the validation

Reply:
I'm going for uniform return, empty list.